src/validators/city-validator.js

```diff
@@ -0,0 +1,31 @@
+const  {check, param}  =  require ( 'express-validator' );
+const {validateResult} = require('../helpers/response-result');
+
+const validationStateId  = [
+    param('stateID')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .matches(/^[a-z0-9]{24}$/).withMessage('El ID debe tener números y letras minúsculas')
+        .isLength(24).withMessage('Debe tener 24 caracteres'),
+    (request, response, next) => {
+        validateResult(request, response, next);
+    }
+]
+
+const validationCity  = [
+    check('nameCity')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isString().withMessage('El campo debe ser una cadena')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
        .isString().withMessage("El campo debe ser una cadena")
```